### DISCUSSION TITLE
levureBuildStandalonesForTesting and mobile

### DISCUSSION THREAD

**Discussion by macMikey** [trevordevore/levure#170] [Q&A] [answered]
How does levureBuildStandalonesForTesting work with mobile apps? If I understand the command correctly, this is a way to sideload beahviors on-the-fly, which is obviously possible on mobile, but wouldn't you need to specify a URL to get the files from?

**Comment by trevordevore** [comment, marked as answer]
It only builds standalone for the desktop. The standalones load the app from the app directory each time you launch so that you can test your current code in a standalone whenever you want without having to package the entire app. IIRC it relies on files being in a predefined location relative to the standalone.
